nixpkgs/repositories.bzl

```diff
@@ -50,14 +51,10 @@ def rules_nixpkgs_dependencies(rules_nixpkgs_name = "io_tweag_rules_nixpkgs"):
     if strip_prefix:
         strip_prefix += "/"
 
-    for name, prefix in [
-        ("rules_nixpkgs_core", "core"),
-        ("rules_nixpkgs_cc", "toolchains/cc"),
-        ("rules_nixpkgs_java", "toolchains/java"),
-        ("rules_nixpkgs_python", "toolchains/python"),
-        ("rules_nixpkgs_go", "toolchains/go"),
-        ("rules_nixpkgs_rust", "toolchains/rust"),
-        ("rules_nixpkgs_posix", "toolchains/posix"),
+    for name, prefix in [("rules_nixpkgs_core", "core")] + [
+        ("rules_nixpkgs_" + toolchain, "toolchains/" + toolchain)
```

Comment:
Perhaps needs a check to guard against non-existent toolchains. Otherwise, users could see very confusing error messages on typos.

Reply:
Good catch! I added a check to error out if an unknown toolchain is passed in.